database/connector.py

```diff
@@ -199,7 +225,9 @@ def insert_honeypot_events_data_from_module_processor(ip, module_name, date, dat
             "module_name": module_name,
             "date": date,
             "data": data,
-            "country": byte_to_str(IP2Location.get_country_short(byte_to_str(ip))),
-            "machine_name": network_configuration()["real_machine_identifier_name"]
+            "country": byte_to_str(IP2Location
+                            .get_country_short(byte_to_str(ip))),
```

Comment:
it's a valid thing but the readability is compromised here

Reply:
@dhirensr I think you are looking at an older commit.